source/administration/statistics.rst

```diff
@@ -0,0 +1,75 @@
+Statistics
+================
+
+Statistics on users, posts and channels are tracked for each system and
+team. Enterprise Editions have access to advanced system statistics.
+
+System Statistics
+-----------------
+
+System statistics are viewable under **System Console** > **Site
+Reports** > **View Statistics**. The data shown here is a cumulative sum
+across all teams on the system.
+
+**Total Users:** The total number of accounts created on your system,
+including both active and inactive accounts.
+
+**Total Posts:** The total number of posts made in all the teams on your
+system, including deleted posts and posts made using automation.
+
+**Public Groups:** The number of public channels created in all the
+teams on your system, including channels that may have been archived.
+
+**Private Groups:** The number of private groups in all the teams on
+your system, including groups that may have been archived.
+
+**Active Users With Posts:** Users who logged in and made a post on a
+certain day in all the teams on your system.
+
+Advanced system statistics (Enterprise)
+^^^^^^^^^^^^^^
+
+Enterprise Edition includes additional system statistics.
+
+**Incoming Webhooks:** The number of active Incoming Webhooks currently
+setup on your system. Webhooks that are created and then removed in the
+Account Settings menu are not counted.
+
+**Outgoing Webhooks:** The number of active Outgoing Webhooks currently
+setup on your system. Webhooks that are created and then removed in the
+Account Settings menu are not counted.
+
+**Channel Types:** This chart displays the number of public channels and
+private groups in a visual format, including channels that may have been
+archived.
+
+**Posts, Files and Hashtags:** This chart displays the number of posts
+containing files, hashtags or text-only. Posts containing both files and
+hashtags are counted in both categories.
+
+Team Statistics
+---------------
+
+Team Statistics are viewable under **System Console** > **Teams** >
+**Statistics**.
+
+**Total Users:** The total number of accounts created on your team,
+including both active and inactive accounts.
+
+**Total Posts:** The total number of posts made in your team, including
+deleted posts and posts made using automation.
+
+**Public Groups:** The number of public channels created by your team,
+including channels that may have been archived.
+
+**Private Groups:** The number of private groups created by your team,
+including groups that may have been archived.
+
+**Active Users With Posts:** Users who logged in and made a post on a
+certain day. Includes system messages posted from the user's account.
+
+**Recently Active Users:** Users that have logged in and had recent
+browser activity in Mattermost.
+
+**Newly Created Users:** Users that have recently completed the sign-up
+process to create a Mattermost account on the team.
```

Comment:
I think this might need to be clarified. ie is it new joins on the team? Since users can now have accounts that aren't connected to teams